askama_shared/src/generator.rs

```diff
@@ -66,7 +67,7 @@ fn build_template(ast: &syn::DeriveInput) -> Result<String, CompileError> {
         &contexts,
         heritage.as_ref(),
         MapChain::new(),
-        config.automatic_block_trim,
+        automatic_block_trim,
```

Comment:
Can we just pass `config.automatic_block_trim` directly, or did pass ownership of the config somewhere?

Reply:
Yes we can! I think at some point it wasn't possible because of how I was trying to parse the derive proc macro. Fixed it,